webapps/src/main/webapp/vue-app/tasks-management/components/tasks/TasksDashboard.vue

```diff
@@ -491,8 +491,7 @@ export default {
       });
     },
     updateTasksList() {
-      const filterTasks = this.filterActive ? this.taskQueryFilter : this.filterTasks;
```

Comment:
does the task filter drawer work with this change ?
this removed check was added for a reason, could you check why was it added ?